presto-docs/src/main/sphinx/connector/pinot.rst

```diff
@@ -0,0 +1,105 @@
+===============
+Pinot Connector
+===============
+
+The Pinot connector allows Presto to query data stored in
+`Apache Pinot™ (Incubating) <https://pinot.apache.org/>`_.
+
+Compatibility
+-------------
+
+The Pinot connector is compatible with all Pinot versions starting from 0.1.0.
+
+Configuration
+-------------
+
+To configure the Pinot connector, create a catalog properties file
+``etc/catalog/pinot.properties`` with atleast the following contents,
+replacing ``host1:9000,host2:9000`` with a comma-separated list of Pinot Controller nodes
```

Comment:
Move this section as full sentence after the section. Also explain if this needs only the FQDN hostname (or IP address) or also a protocol like https:// or whatever.